Update README.md (#23)
fiboa/cli: README.md

# fiboa CLI

> In order to make working with fiboa easier we have developed command-line interface (CLI) tools such as validation and file format conversion. To get started with the fiboa CLI please follow the instructions below:

Reviewer: ```suggestion
In order to make working with fiboa easier we have developed command-line interface (CLI) tools such as validation and file format conversion.
You can find instuctions how to get started below.
```

## Installation

> You will need **Python 3.9+** installed. 

Reviewer: ```suggestion
You will need to have **Python 3.9** or any later version installed. 
```
